packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -231,7 +231,8 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
 
       this.setState({
         selectedIndices: indices,
-        currentOptions: newProps.options
+        currentOptions: newProps.options,
+        suggestedDisplayValue: undefined
```

Comment:
I think that we should only do this if null is passed in. Otherwise the suggested value should be whatever it would be normally.

Reply:
I don't follow totally. null is passed in to what?

Reply:
Read the issue. Updated the code to get this going.